core/domain/suggestion_registry.py

```diff
@@ -187,18 +212,38 @@ def validate(self):
         """Validates a suggestion object of type
         SUGGESTION_TYPE_EDIT_STATE_CONTENT.
 
-        Returns:
-            bool. The validity of the suggestion object.
+        Raises:
+            ValidationError: One or more attributes of the
+                SuggestionEditStateContent object are invalid.
         """
         super(SuggestionEditStateContent, self).validate()
 
+        if (
+                self.score_category.split(
```

Comment:
This is ugly, especially since it's repeated in line 229-230 as well. Better to define helper variables for the two parts of the string.

Reply:
Added helpers to get the parts separately and also tests for them.